web/src/styles/utilities.css

```
@@ -98,153 +98,179 @@
+      #000000d9 calc(var(--cds-scroll-fade-top) * 0.87),
+      #000 var(--cds-scroll-fade-top),
+      #000 calc(100% - var(--cds-scroll-fade-bottom)),
+      #000000d9 calc(100% - var(--cds-scroll-fade-bottom) * 0.87),
+      #000000b3 calc(100% - var(--cds-scroll-fade-bottom) * 0.75),
+      #00000080 calc(100% - var(--cds-scroll-fade-bottom) * 0.6),
+      #0000004d calc(100% - var(--cds-scroll-fade-bottom) * 0.45),
+      #00000026 calc(100% - var(--cds-scroll-fade-bottom) * 0.3),
+      #0000000d calc(100% - var(--cds-scroll-fade-bottom) * 0.15),
+      transparent
+    );
+    -webkit-mask-image: var(--cds-scroll-fade-mask);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)